test-projects/basic/server.js

```diff
@@ -53,21 +53,31 @@ function dropAllDatabases(adapters) {
 }
 
 /*
- * Seed the intial data using `createItem` API
- * 1. Insert all the lists having no associated relatioships: User, PostCategory, and ReadOnlyList.
- * 2. Insert the `Post` data, with the required relationships, via `connect` nested mutation.
+ * Seed the initial data using `createItem` API
+ * 1. Insert all the lists having no associated relationships: User, PostCategory, and ReadOnlyList.
+ * 2. Insert `Post` data, with the required relationships, via `connect` nested mutation.
  */
 async function seedData(initialData) {
-  const [[{ createUsers: users }], [{ createPostCategories: postCategories }]] = await Promise.all(
-    ['User', 'PostCategory', 'ReadOnlyList'].map(listName =>
-      createItems({
-        keystone,
-        listName,
-        items: initialData[listName],
-        returnFields: listName === 'User' ? 'id, email' : 'id, name',
-      })
-    )
-  );
+  const { createUsers: users } = await createItems({
```

Comment:
`createItems` uses `runChunkedMutation` which returns an Array of resolved promises. So, the line should be 
```js
const [{createUsers: users}] = await createItems(...);
```